src/storage/leveldb/wallet_index.ts

```diff
@@ -53,6 +53,9 @@ export default class LevelWalletIndex implements IKVWalletIndex {
    * @param {number} value The value to set.
    */
   async _setNumber(dest: 'access'|'wallet', key: string, value: number) {
+    if (value < 0) {
+      return;
```

Comment:
Shouldn't we throw here, instead of silently ignoring the call?

Reply:
This is to avoid errors on the tests, but maybe we should let this throw and make checks before this happens.

Reply:
I agree it would be better to have the assert and exception here. However, it would affect many other functions not related with theme of the PR. I suggest to make this change in another PR.